_docs-sources/guides/build-it-yourself/achieve-compliance/deployment-walkthrough/manual-steps.md

```diff
@@ -73,28 +73,24 @@ You can also configure an automated system integration if you have a third party
 Follow the steps in the [AWS
 docs](https://docs.aws.amazon.com/sns/latest/dg/sns-http-https-endpoint-as-subscriber.html) on how to add an HTTPS endpoint as a subscriber to the alerts.
 
-## 6. Manually maintain buckets to analyze in the `buckets_to_analyze` variable
+## 6. Add existing buckets to a new Macie classification job.
 
 Macie is a new AWS service that uses machine learning (ML) and pattern matching to discover and help protect your sensitive
-data. To set up Macie to analyze the desired S3 buckets, you’ll need to create a **Macie classification job**. Typically,
-you’ll want it to analyze all the buckets in the region. However, the terraform AWS provider does not support specifying
-all the buckets in a region - it requires that an explicit list of buckets be provided. Therefore, you’ll
-need to maintain an explicit list of buckets per region, namely in the variable `buckets_to_analyze`. This list of buckets needs to be maintained in the future as new buckets are created in the account. Please read the
-[documentation](https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/blob/master/modules/security/macie/variables.tf#L21-L30)
-for this variable in order to understand how to structure the list of buckets per region.
+data. The Landing Zone solution already configured Macie in all deployed accounts. The last step is to setup S3 buckets to be analyzed.
```

Comment:
```suggestion
data. The Landing Zone solution already configured Macie in all deployed accounts. The last step is to specify the S3 buckets to be analyzed.
```

Reply:
What does the CIS 1.5 benchmark recommend? Do you configure _all_ S3 buckets to be analyzed? Or just some? What does the Steampipe benchmark verify?

If it's all buckets, we need our instructions to say this clearly and explicitly, and explaine how to do this in each AWS account and each AWS region.

Reply:
>What does the CIS 1.5 benchmark recommend? Do you configure all S3 buckets to be analyzed? Or just some? >What does the Steampipe benchmark verify?

The recommendation says:
>  >Select the check box for each bucket that you want Macie to analyze as part of the job

So not all buckets need to be analyzed. The Steampipe benchmark will fail if not _all_ buckets are being analyzed.

>If it's all buckets, we need our instructions to say this clearly and explicitly, and explaine how to do this in each AWS account and each AWS region.

Gotcha. Macie is a global resource, so at least this is once per account.

I will update the docs!